Marlin/Configuration_adv.h

```diff
@@ -3669,6 +3669,18 @@
     #define LASER_TEST_PULSE_MIN           1   // Used with Laser Control Menu
     #define LASER_TEST_PULSE_MAX         999   // Caution: Menu may not show more than 3 characters
 
+   /**
+    * Laser Safety Timeout
+    *
+    * If no head movement for this period then the laser will turn off.
+    * Consider material flammability, cut rate and G-code order when
+    * setting value.  Too low and it could turn off during a very slow move,
+    * too high a value and the material may ignite before the laser is turned off.
+    *
+    * Value must be < (DEFAULT_STEPPER_DEACTIVE_TIME * 1000)
```

Comment:
The stepper deactivation time can be changed at runtime with `M18 S<sec>`, so this is not a meaningful requirement. You'll have to adjust the code to deal with this fluid situation.